components/global/core/io.cellery.observability.api/src/main/java/io/cellery/observability/api/interceptor/CORSInterceptor.java

```diff
@@ -34,13 +34,14 @@
  * which methods are allowed. Therefore an endpoint with HTTP Method OPTIONS should be added to all services.
  */
 public class CORSInterceptor implements RequestInterceptor {
+
     @Override
     public boolean interceptRequest(Request request, Response response) {
         response.setHeader(Constants.ACCESS_CONTROL_ALLOW_METHODS, HttpMethod.GET + "," + HttpMethod.POST +
                 "," + HttpMethod.PUT + "," + HttpMethod.DELETE);
         response.setHeader(Constants.ACCESS_CONTROL_MAX_AGE, Constants.MAX_AGE);
-        response.setHeader(Constants.ACCESS_CONTROL_ALLOW_HEADERS, HttpHeaders.CONTENT_TYPE);
-
+        response.setHeader(Constants.ACCESS_CONTROL_ALLOW_HEADERS,
+                HttpHeaders.CONTENT_TYPE + "," + HttpHeaders.AUTHORIZATION);
```

Comment:
Why do we set multiple headers at once ?. Also in line 42 also we set a header.

Reply:
This actually sets multiple values (seperated by comma) for a single header type.